tests/run-make/glibc-symbols-x86_64-unknown-linux-gnu/rmake.rs

```diff
@@ -0,0 +1,105 @@
+// Check that the compiler (rustc) is not using newer glibc symbols than a specified minimum.
+
+//@ only-x86_64-unknown-linux-gnu
+//@ ignore-cross-compile
```

Comment:
Shouldn't this test also be ignored for local builds? For local builds we don't compile with an old glibc version and thus may use arbitrarily new symbol versions.

Reply:
Yeah, that's why this is still a draft, as I noted in the PR description, I need to teach compiletest to run certain tests only in opt-dist tests, post-optimization on CI.